docs/health/health_design/design.rst

```diff
@@ -1,6 +1,63 @@
 Design
 ======
 
+The previous page showed you how to use pyATS Health Check easily. This page is for those who want to use their own pyATS Health Check. If you are happy with previous page way, please move on :ref:`processor key in YAML<processor_key_in_yaml>`.
+
+Collect and monitor the state of your devices as your testscript is executing with pyATS Health Check. It can collect traceback, core files, etc. pyATS Health Check can be even yaml driven and it is based on :ref:`Blitz<blitz>`. All Blitz functionalities are fully supported in health check.
```

Comment:
```suggestion
You can collect and monitor the state of your devices as your testscript is executing with pyATS Health Check. It can collect traceback, core files, etc. pyATS Health Check is yaml driven and it is based on :ref:`Blitz<blitz>`. All Blitz functionalities are supported in Health Check.
```